internal/setup/webhooks.go

```diff
@@ -57,51 +58,93 @@ func ManageCerts(mgr ctrl.Manager, setupFinished chan struct{}, restartOnSecretR
 
 // createWebhooks creates all mutators and validators.
 func createWebhooks(mgr ctrl.Manager, f *forest.Forest, opts Options) {
+	decoder := admission.NewDecoder(mgr.GetScheme())
+
+	// NOTE(ryotarai): The injecting mechanism is removed in https://github.com/kubernetes-sigs/controller-runtime/pull/2134
+	// For now, the decoder and client are injected manually, but we might want to replace this with sigs.k8s.io/controller-runtime/pkg/builder.WebhookManagedBy
```

Comment:
👀 May I ask you to create the issue for addressing this comment?

Reply:
Created an issue for this: https://github.com/pfnet/hierarchical-namespaces/issues/20